### PR TITLE
Sector check command

### PR DESCRIPTION
**Pull request by magik6k**

### CODE REVIEW

**Comment by magik6k** [comment]
(fixed up https://github.com/filecoin-project/lotus/pull/5013)

**Review by magik6k — COMMENTED** [review]
LGTM with my changes (which should get another review)

**Comment by magik6k** [comment]
(tested on my miner)

File: storage/wdpost_run_test.go [hunk 1]
```diff
@@ -125,9 +125,9 @@ func (m *mockProver) GenerateWindowPoSt(ctx context.Context, aid abi.ActorID, si
 type mockFaultTracker struct {
 }
 
-func (m mockFaultTracker) CheckProvable(ctx context.Context, pp abi.RegisteredPoStProof, sectors []storage.SectorRef) ([]abi.SectorID, error) {
+func (m mockFaultTracker) CheckProvable(ctx context.Context, pp abi.RegisteredPoStProof, sectors []storage.SectorRef) (map[abi.SectorID]string, error) {
 	// Returns "bad" sectors so just return nil meaning all sectors are good
```
Comment (dirkmc) [inline storage/wdpost_run_test.go:129]:
```suggestion
	// Returns "bad" sectors so just return empty map meaning all sectors are good
```

**Review by dirkmc — APPROVED** [review]
One nit but otherwise LGTM 👍

**Review by vyzo — APPROVED** [review]
LGTM

**Review by Kubuxu — APPROVED** [review]

**Pull request merged**